eclair-core/src/main/scala/fr/acinq/eclair/remote/EclairInternalsSerializer.scala

```diff
@@ -129,6 +130,8 @@ object EclairInternalsSerializer {
     (path: String) => system.provider.resolveActorRef(path),
     (actor: ActorRef) => Serialization.serializedActorPath(actor))
 
+  def typedActorRefCodec[T](system: ExtendedActorSystem):Codec[typed.ActorRef[T]] = actorRefCodec(system).xmap(_.toTyped, _.toClassic)
+
```

Comment:
[According to akka devs](https://discuss.lightbend.com/t/akka-typed-serialization/4336), this should be instead:

```suggestion
  def typedActorRefCodec[T](system: typed.ActorSystem[Nothing]): Codec[typed.ActorRef[T]] = variableSizeBytes(uint16, utf8).xmap(
    (path: String) => ActorRefResolver.get(system).resolveActorRef(path),
    (actor: typed.ActorRef[T]) => ActorRefResolver.get(system).toSerializationFormat(actor))
```

NB: untested, and needs to be called with `system.toTyped`

Reply:
Note to self: this will require synchronous update of `eclair-front`